zhaquirks/sonoff/s60zbtpf.py

```diff
@@ -0,0 +1,190 @@
+"""Sonoff S60ZBTPF - Zigbee Smart Plug."""
+
+from typing import Final
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfPower,
+)
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl import ClusterType, foundation
+import zigpy.zcl.foundation as zcl_f
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class SonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        # network_led = ZCLAttributeDef(
+        #     name="network_led",
+        #     id=0x0001,
+        #     type=t.Bool,
+        # )
+        outlet_control_protect_setting = ZCLAttributeDef(
+            name="outlet_control_protect_setting",
+            id=0x7007,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload_enable = ZCLAttributeDef(
+            name="ac_current_max_overload_enable",
+            id=0x700C,
+            type=t.uint8_t,
+        )
+        ac_voltage_max_overload_enable = ZCLAttributeDef(
+            name="ac_voltage_max_overload_enable",
+            id=0x700E,
+            type=t.uint8_t,
+        )
+        ac_power_max_overload_enable = ZCLAttributeDef(
+            name="ac_power_max_overload_enable",
+            id=0x7010,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload = ZCLAttributeDef(
+            name="ac_current_max_overload",
+            id=0x700D,
+            type=t.uint32_t,
+        )
+        ac_voltage_max_overload = ZCLAttributeDef(
+            name="ac_voltage_max_overload",
+            id=0x700F,
+            type=t.uint32_t,
+        )
+        ac_power_max_overload = ZCLAttributeDef(
+            name="ac_power_max_overload",
+            id=0x7011,
+            type=t.uint32_t,
+        )
+
+    class ServerCommandDefs(zcl_f.BaseCommandDefs):
+        """Server command definitions."""
+
+        self_test: Final = zcl_f.ZCLCommandDef(
+            id=0x00, schema={"identify_time": t.uint8_t}, direction=False
+        )
+
+    async def _read_attributes(
+        self,
+        attribute_ids: list[t.uint16_t],
+        *args,
+        manufacturer: int | t.uint16_t | None = None,
+        **kwargs,
+    ):
+        """Read attributes ZCL foundation command."""
+        return await super()._read_attributes(
+            attribute_ids,
+            *args,
+            manufacturer=foundation.ZCLHeader.NO_MANUFACTURER_ID,
+            **kwargs,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
```

Comment:
We should be able to set `manufacturer_id_override` for the cluster instead.
See as an example: https://github.com/zigpy/zha-device-handlers/blob/a941a8968ece77df01defe8bbee337c939e04b82/zhaquirks/sonoff/swv.py#L24